tests/integ/test_column.py

```diff
@@ -216,3 +216,86 @@ def test_column_with_builtins_that_shadow_functions(session):
     with pytest.raises(TypeError) as ex_info:
         TestData.double1(session).select(sum(col("a"))).collect()
     assert iter_error_msg_text in str(ex_info)
+
+
+@pytest.mark.parametrize("enable_sql_simplifier", [True, False])
+def test_column_regex(session, enable_sql_simplifier, local_testing_mode):
+    if not enable_sql_simplifier and local_testing_mode:
+        pytest.skip("disable sql simplifier is not supported in local testing mode")
```

Comment:
```suggestion
    if not session.sql_simplifier_enabled and local_testing_mode:
        pytest.skip("disable sql simplifier is not supported in local testing mode")
```
we can use this to do this check